module/exporter-shopware-6/src/Infrastructure/Connector/Shopware6Connector.php

```diff
@@ -21,10 +21,7 @@ class Shopware6Connector
 
     private ?string $token;
 
-    /**
-     * @var \DateTimeInterface
-     */
-    private $expiresAt;
+    private \DateTimeInterface $expiresAt;
```

Comment:
Should we use `\DateTimeInterface` or `DateTimeInterface` and declaration of `DateTimeInterface` in `use`?

Reply:
I don't think we have a standard here. I personally prefer using `\DataTimeInterface` because we right away see the usage of global classes but that's just me. We can think of standardization potentially.